dependencies/build_imagemagick.sh

```
@@ -52,20 +48,43 @@ curl -fsSL --retry 2 --retry-delay 5 --retry-max-time 60 -o ./magick.tar.gz \
+  --with-zstd
+)
 
 ./configure \
   --enable-64bit-channel-masks \
-  --enable-static --enable-shared --enable-delegate-build \
-  --without-x --without-magick-plus-plus \
-  --without-perl --disable-doc \
+  --enable-static \
+  --enable-shared \
+  --enable-delegate-build \
+  ${FEATURES[@]} \
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)